src/Agent.Sdk/DockerVersion.cs

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System;
+
+namespace Agent.Sdk
+{
+    public class DockerVersion
+    {
+        public DockerVersion()
+        {
+
+        }
+
+        public DockerVersion(Version serverVersion, Version clientVersion)
+        {
+            this.ServerVersion = serverVersion;
+            this.ClientVersion = clientVersion;
+        }
+
+        public Version ServerVersion { get; set; }
```

Comment:
Can we change these to `private set`?

Reply:
this was just a code relo, soooo maybe, but not with this PR